### PR TITLE
added kvrocks recipe

### PR DESCRIPTION
**Pull request by mguijarr**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mguijarr** [comment]
@conda-forge/staged-recipes, ready for review!

File: recipes/kvrocks/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{% set name = "kvrocks" %}
+{% set version = "2.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # If getting the source from GitHub, remove the line above,
+  # uncomment the line below, and modify as needed. Use releases if available:
```
Comment (xhochy) [inline recipes/kvrocks/meta.yaml:10]: Please remove the comment
```suggestion
```

File: recipes/kvrocks/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  url: https://github.com/apache/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 8883a7d05f604098d92cda84128e919dd513540bdd5efb31da69b366e5c7fec1
+
+build:
+  number: 0
+  skip: True  # [win or win_64 or osx]
```
Comment (xhochy) [inline recipes/kvrocks/meta.yaml:16]:
```suggestion
  skip: True  # [win or osx]
```

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/kvrocks/meta.yaml (hunk 1 above)
Comment (mguijarr) [reply recipes/kvrocks/meta.yaml]: Done

**Comment by mguijarr** [comment]
@xhochy thanks for the review - made requested changes

**Review by xhochy — APPROVED** [review]

**Pull request merged**